src/util/constants.ts

```diff
@@ -56,3 +56,15 @@ export const BEHAVIOR_TYPES = [
 ];
 
 export const DISPLAY = ":99";
+
+export enum ExitCodes {
+  Success = 0,
+  GenericError = 1,
+  Failed = 9,
+  OutOfSpace = 3,
+  BrowserCrashed = 10,
+  InterruptedGraceful = 11,
+  InterruptedImmediate = 13,
+  Fatal = 17,
```

Comment:
Looking through the code, a lot of the cases where we use this actually match against the other status codes. Maybe we don't need a generic `Fatal` exception, or should only use it as a fallback if there's a case where there's not a more specific status code to use?

Reply:
Maybe this is better handled in #584, on second thought

Reply:
Yeah, I think we can add additional statuses via #584, keeping this for backwards compatibility for now.